exporter.py

```diff
@@ -183,6 +184,11 @@ def __init__(self, ccu_host, ccu_port, auth, gathering_interval, reload_names_in
         self.gathering_interval = int(gathering_interval)
         self.reload_names_interval = int(reload_names_interval)
         self.devicecount = Gauge('devicecount', 'Number of processed/supported devices', labelnames=['ccu'], namespace=self.METRICS_NAMESPACE)
+        # Upon request export the seconds since the last successful update.
+        # This is robust against internal crashes and can be used by the healthcheck.
+        self.refresh_time = time.time()
```

Comment:
Why do you again set the current time here? This shadows the time set in the last loop (at line 155). The delta calculated in line 191 will always be near 0 by this. 
I'd suggest to remove this, then the metric shows the time since the last successful loop execution.

Reply:
Note that this is not in the loop, but only the `__init__`. It's executed just once.

We do however have to initialize `self.refresh_time` at something, and initializing it at the startup time felt like the best compromise.
Alternatives I considered and decided agaist:
* `self.refresh_time = None` makes the callback more compliated, as it would have to `if/else` and return `float("nan")` when invoked until the first successful loop completion.
* `self.refresh_time = 0` would result in extremely large values for the metric until the first loop completed.

The metric shows the age since the last successful refresh, as intended.
Here's a screenshot from an event a few days ago:

![image](https://github.com/sfudeus/homematic_exporter/assets/5894642/e148388b-e911-4bb9-ac2a-92a5b3cc5e26)

In this case my Raspberry Pi Zero dropped out of the WiFi, and an Airflow DAG that runs every 10 minutes power-cycled it via a remote switch. So this wasn't the kind of event I added the metric for, but it still shows that it works :)

Reply:
Oh, yes thanks. I in fact missed that this is during init. 
lgtm then. 
Let me try this a bit in my k8s cluster and see what it does there. I can use the healthcheck as a liveness probe. 

One more thing: 
Let's rename the metric to refresh_age_seconds (adding the unit is considered best practice for prometheus metrics.